### ISSUE TITLE
When will Luo be self-compiling?

### ISSUE THREAD

**Issue by InDieTasten** [oeed/Luo#4]
In #3 you mentioned, that you are writing the compiler in swift, because the swift type system is what you are attempting to work towards for Luo.

This implies, that the compiler currently only runs using swift, and cannot in of itself be compiled to plain Lua.

Is there a rough idea as to when this will happen?

I would assume, there might be some community involvement going on once this barrier would come down, which is why I'm asking. I'm a huge TypeScript fan and would totally love to see something like this for Lua, although I'm not familiar with swift at all and it's been a while since I've used lua the last time.

---
Basically feeding you with more readme material here ;D

**Comment by oeed** [comment]
I know this isn't really what you want to hear, but it'll be done when it's done. Once the Swift compiler is done I might first make a test program in it just to see if it's missing anything, but once I am happy with the language the top priority is getting it written in Luo. I'm not setting any timeframes though, I'm simply working on this in my free time.
